docs/registry/_index.md

```diff
@@ -96,87 +102,28 @@ Once you have added and configured the module you would like to use in the Viam
 
 `viam-server` manages the [dependencies](/internals/rdk/#dependency-management), [start-up](/internals/rdk/#start-up), [reconfiguration](/internals/rdk/#reconfiguration), [data management](/services/data/capture/#configure-data-capture-for-individual-components), and [shutdown](/internals/rdk/#shutdown) behavior of your modular resource.
 
-### Tutorials using modules
```

Comment:
should those be still kept somewhere? Or instead should we have the how to page here?

Reply:
nvm I agree